csharp/src/Google.Protobuf.Test/GeneratedMessageTest.Proto2.cs

```diff
@@ -261,6 +261,16 @@ public void RequiredFields()
             Assert.True(message.IsInitialized());
         }
 
+        // Code was accidentally left in message parser that threw exceptions when missing required fields after parsing.

```

Comment:
thanks for adding the comment.
nit: Use `/// <summary>` block for comments outside the method body.